src/components/cylc/Drawer.vue

```diff
@@ -51,6 +51,16 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
         </v-list-item-action>
         <v-list-item-title>Dashboard</v-list-item-title>
       </v-list-item>
+      <v-list-item
+          to="/graphiql"
+          active-class="primary grey--text text--darken-3"
+          class="v-list-item"
+      >
+        <v-list-item-action>
+          <v-icon>mdi-graphql</v-icon>
```

Comment:
Looks like a graph, not sure if we could use this in some other place.

Also, I placed the link under Dashboard, but perhaps it should go somewhere else?

Reply:
Good question about where to put it.

I would have thought it would be best implemented as a cylc "view", even though graphiql isn't actually a view of a single workflow we could pre-populate the query with:

```
query {
    workflows (ids: [<workflowid>] {
        id
    }
}
```

Reply:
That's what I suggested below too, and this pre-populate idea seems an easy way to make that work (in terms of displaying the right workflow, if not being driven by the common subscription).

Reply:
That might be doable. But first I would prefer to replace the current GraphiQL from UIS by this one.

What about:

- create `/$user/#/graphiql` view that shows GraphiQL in full screen
- remove GraphiQL from UIS
- remove "Close" and change by "Related to #352" from this issue
- create follow up for #352 to component-ize the GraphiQL, where we can discuss where to add it, how to load data, etc?

That way we get the nice error messages and the updated GraphiQL in Cylc UI.